### PR TITLE
System Backend Documentation

### PR DESCRIPTION
**Pull request by choxx**

### CODE REVIEW

File: docs/tech/system/backend.md [hunk 1]
```diff
@@ -10,12 +10,145 @@ Create diagram of their interaction
 Database structure
```
Comment (karntrehan) [inline docs/tech/system/backend.md]: Can remove this line?

File: docs/tech/system/backend.md [hunk 2]
```diff
@@ -10,12 +10,145 @@ Create diagram of their interaction
...
+
+- TSP(Total Studnet Passing)
+- Boolean
+- Marks
+
+There are a total of 32 possible combinations from the above divisions and out of those assessments, 
+we support the Grade and Marks evaluation parameter, LO, Unit, Questions, None assessment type, 
+and TSP outcome.
+
+#### Assessment Creation
```
Comment (karntrehan) [inline docs/tech/system/backend.md:67]: Can we link this to the code base as well? Same for other modules as well.

File: static/img/assessment-create.png
Comment (karntrehan) [inline static/img/assessment-create.png:1]: BEAUTIFUL!

File: static/img/samarth-hp-backend-Student-Enrollment.png
Comment (karntrehan) [inline static/img/samarth-hp-backend-Student-Enrollment.png:1]: Lets add all the raw source draw.io files to the engineering drive. https://drive.google.com/drive/folders/156H6nI1P-GgQ35Ax9WoXj3pVtFM3oCY3?usp=sharing

**Review by karntrehan — APPROVED** [review]

File: docs/tech/system/backend.md (hunk 1 above)
Comment (choxx) [reply docs/tech/system/backend.md]: Done.

File: docs/tech/system/backend.md (hunk 2 above)
Comment (choxx) [reply docs/tech/system/backend.md:67]: Done.

File: static/img/samarth-hp-backend-Student-Enrollment.png
Comment (choxx) [reply static/img/samarth-hp-backend-Student-Enrollment.png:1]: Done.

**Pull request merged**